tests/test_y_logging.py

```diff
@@ -137,6 +137,72 @@ def test_mlflow_pickle():
     testing_utils.clear_save_dir()
 
 
+def test_comet_logger():
+    """
+    verify that basic functionality of Comet.ml logger works
+    """
+    reset_seed()
+
+    try:
+        from pytorch_lightning.logging import CometLogger
+    except ModuleNotFoundError:
+        return
+
+    hparams = testing_utils.get_hparams()
+    model = LightningTestModel(hparams)
+
+    # API key for dummy Comet.ml account
+    logger = CometLogger(
+        api_key="KnmgASRHHyxWXOpwUfgrAFz8C",
```

Comment:
I'm not familiar with CometML, so could you explain what you did here? Is this the key for a dummy account that you created manually?

Reply:
After a quick glance through the docs, it seems like maybe `OfflineExperiment` is what we'll want for tests.

Reply:
If we want to test using OfflineExperiment then we'll need to add that as an option in the CometLogger class.

What do you think about setting it up such that you either pass in an API key or a directory, and having that determine if it saves in online/offline mode? I think I'd give priority to online mode if both an API key and a directory were passed.

Reply:
Yeah, I think that makes sense. It will be good to have the offline option, in case someone is running behind a firewall or something.

Reply:
Great! I'll add that option and then update the docs